style.css

```diff
@@ -1,34 +1,204 @@
 * {
   margin: 0;
   padding: 0;
+}
+:root {
+  --home-bg-color: #99dfff;
+  --home-primary-color: #43c4ff;
+  --home-secondary-color: #0095da;
+  --black-color: #000000;
+  --white-color: #ffffff;
+}
+
+/* body {
+  background-image: linear-gradient(
+    140deg,
+    rgb(255, 255, 255),
+    var(--home-bg-color)
+  );
+  background-repeat: no-repeat;
+} */
+
+.main-container {
+  height: 100vh;
+  width: 100%;
+  padding: 0 5%;
   box-sizing: border-box;
+  position: relative;
 }
-.header {
-  height: 8rem;
+
+.navbar {
+  width: 100%;
+  height: 15vh;
   display: flex;
-  flex-wrap: wrap;
-  /* background-color: rgb(222, 228, 226); */
+  align-items: center;
 }
-.header__nav {
-  /* background-color: aquamarine; */
-  flex-grow: 5;
-  height: 100%;
+
+.logo {
+  width: 2rem;
+  cursor: pointer;
+  margin-left: 4rem;
+  font-size: 2.5rem;
+  color: var(--home-secondary-color);
+}
+
+nav {
+  margin-left: auto;
+}
+
+nav ul {
   display: flex;
-  justify-content: center;
-  align-items: center;
 }
-.header__nav nav ul {
+
+nav ul li {
+  list-style: none;
+  margin: 0 1rem;
+}
+
+nav ul li a {
+  text-decoration: none;
+  font-size: 1.4rem;
+  color: var(--black-color);
+}
+
+nav ul li a:hover {
+  color: var(--home-secondary-color);
+}
+
+.navsidebar {
+  width: 60%;
+  height: 100vh;
+  top: 0;
+  padding-top: 5rem;
+  position: absolute;
+  left: -200rem;
   display: flex;
+  flex-direction: column;
+  align-items: center;
+  transition: all 1s;
+  z-index: 2;
+  background-color: var(--home-primary-color);
```

Comment:
Here as there are a lot of properties in the same class, you could sort them in ascending alphabetical order as that helps in debugging and maintaining the code.